atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -41,5 +41,14 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
         return new ObjectMapper().convertValue(convertedMap, Event.class);
     }
 
+    static Map<Keyword, Object> toKeywordMap(Event event) {
+        Map<String, Object> stringToObject = new ObjectMapper().convertValue(event, Map.class);
+        Map<Keyword, Object> convertedMap = new HashMap<>();
+        EntryStream.of(stringToObject)
```

Comment:
I'm with you. Is does this in the `toMap` call? Well, I guess if we ever do need `null` values inside `Entry` objects our wonderful tests will catch this